capstone/src/main/webapp/form.html

```diff
@@ -9,10 +9,21 @@
     <meta name="google-signin-client_id" content="">
   </head>
   <body>
+
+    <div id="auth"></div>
+
     <div id="content">
       <p><h1>Quiz</h1></p>
-      <iframe src="https://docs.google.com/forms/d/e/1FAIpQLScrpUcq5Jnv6knV6K3aexWa1m972EmMzHJUvXulHCfBsJaEpw/viewform?embedded=true">Loading…</iframe>
-      <a href="https://docs.google.com/forms/d/1JAOSyrHmloNFiUibqhMLRQj1qcDDhHr0-nDhZ2ouQJM/edit"> For Teacher </a> 
+      <button onclick="callScriptFunction()">Create Form</button>
     </div>
+
+    <pre id="content"></pre>
+    <script src="config.js"></script>
+    <script src="authenticate.js"></script>
+    <script src="form-loader.js"></script>
+    <script async defer src="https://apis.google.com/js/api.js" onload="handleClientLoad()"> </script>
```

Comment:
Having the client of authenticate.js independently need to bring in and have the right gapi tag is not ideal. Instead have the authenticate.js append the script tag within it. I believe something to this effect would work: document.insertChildAfter(document.currentScript, *create the script tag and put here*)

Reply:
Got it.